### PR TITLE
test/system/tests/print.sh: Add

### PR DESCRIPTION
**Pull request by harisokanovic**
Verify tpm2_print works with a sample TPMS_ATTEST quote file.

Signed-off-by: Haris Okanovic <haris.okanovic@ni.com>

### CODE REVIEW

File: test/system/tests/print.sh [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+        tpm2_print -t TPMS_ATTEST -f "$testFileBasename.TPMS_ATTEST" >"$PRINT_TEMP_FILE" 2>&1
+        diff "$PRINT_TEMP_FILE" "$expectedFile"
+    else
+        echo 1>&2 "Could not find input file"
+        exit 1
+    fi
+done
+
+rm "$PRINT_TEMP_FILE"
```
Comment (williamcroberts) [inline test/system/tests/print.sh]: you can just set up an on exit handler to remove this.

File: test/system/tests/print.sh [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+# ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF
+# THE POSSIBILITY OF SUCH DAMAGE.
+#;**********************************************************************;
+
+onerror() {
+    echo "$BASH_COMMAND on line ${BASH_LINENO[0]} failed: $?"
+    if [ -e "$PRINT_TEMP_FILE" ]; then
+        cat "$PRINT_TEMP_FILE"
+        rm "$PRINT_TEMP_FILE"
+    fi
```
Comment (williamcroberts) [inline test/system/tests/print.sh]: This should be in your onexit handler

File: test/system/tests/print-files/quote.printed [hunk 3]
```diff
@@ -0,0 +1,20 @@
+magic: ff544347
```
Comment (williamcroberts) [inline test/system/tests/print-files/quote.printed]: This and the binary blob should go under test fixtures: test/system/fixtures.

However, the test is missing that we want to detect if/when the binary blob output changes and breaks print. I would just output that blob for the test, and run print on it, and use a python script to load the yaml and verify what you can, ie that certain fields exist, etc.

Comment (harisokanovic) [reply test/system/tests/print-files/quote.printed]: Sure, I can add a case to grab a quote and validate it. However, I'd like to have at least one premade blob with known good value -- I.e. a quote blob who's digest was validated by hand in a hexdump.

Comment (williamcroberts) [reply test/system/tests/print-files/quote.printed]: That sounds good.

**Comment by williamcroberts** [comment]
@harisokanovic have you updated this at all or am I missing something?

**Comment by harisokanovic** [comment]
@williamcroberts, I've been busy with other work the last few weeks. I'll post an update next chance I get. Feel free to close this PR if you need to clear it from your view. I can open a new one after I implement your suggestions.

**Pull request closed without merging**